pkg/clients/rds/rds.go

```diff
@@ -131,6 +132,71 @@ func GenerateCreateDBInstanceInput(name, password string, p *v1beta1.RDSInstance
 	return c
 }
 
+// GenerateRestoreDBInstanceInput from RDSInstanceSpec
+func GenerateRestoreDBInstanceInput(name, password string, p *v1beta1.RDSInstanceParameters) *rds.RestoreDBInstanceFromS3Input {
+	c := &rds.RestoreDBInstanceFromS3Input{
```

Comment:
maybe leave a short comment/note inside `GenerateCreateDBInstanceInput` that any changes there should be applied in `GenerateRestoreDBInstanceInput`. It makes it more likely to pick up a bug either during implementation or code review.

Reply:
Added a comment to both functions.